### PR TITLE
Implements #26: Add info command

### PR DESCRIPTION
**Pull request by garnett**
Implements https://github.com/mas-cli/mas/issues/26

I was a bit sloppy and haven't added all proposed fields, only stuff I've found personally useful:

```
> mas info 747648890

Telegram 2.27 [Free]
By: TELEGRAM MESSENGER LLP
Released: 11 Oct 2016
Minimum OS: 10.8
Size: 8.7 MB
From: https://itunes.apple.com/us/app/telegram/id747648890?mt=12&uo=4
```

### CODE REVIEW

**Comment by rodionovd** [comment]
Yay! Will look into this, thanks 🌟

File: mas-cli/Commands/Info.swift [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        return InfoOptions(appId: appId)
+    }
+
+    static func evaluate(_ m: CommandMode) -> Result<InfoOptions, CommandantError<MASError>> {
+        return create
+            <*> m <| Argument(usage: "the app id to show info")
+    }
+}
+
+private struct AppInfoFormatter {
```
Comment (Aghassi) [inline mas-cli/Commands/Info.swift]: Could you pull this out into a separate swift class? I just made an enhancement request about release notes, and I think I can implement it using this class by adding a `formatUpdateNotes` function of sorts. Then just throw that into the outdated command.

Comment (rodionovd) [inline mas-cli/Commands/Info.swift]: @Aghassi the thing is that I'm still not sure what kind of metadata should be displayed here (e.g. what about an app description, ratings, etc). Maybe it'd be even better to let the user decide what they want. This is basically the only reason why this PR is not merged yet.

Comment (Aghassi) [inline mas-cli/Commands/Info.swift]: In that case, maybe we add a flag? Something like `--simple`, which would be the default and give some basic stuff (description, rating, size). Then, another flag like `--detailed` that gives more (author, description, rating, size, minimum OS etc.). Finally, there could be a "custom" flag where the user would have to pass in specific fields? That one seems a tad tedious and not so useful though. Just brainstorming a bit, none of these are right or wrong.

Comment (Aghassi) [inline mas-cli/Commands/Info.swift]: @garnett Any update on this?

Comment (CNG) [inline mas-cli/Commands/Info.swift]: My (admittedly unsolicited) two cents:

Default to simple and flag `--verbose` to show all available fields. I would guess anyone truly needing a custom display could `grep` the output.

Of course you can do more, but releasing something simple is better than releasing nothing due to indecision!

**Comment by phatblat** [comment]
Thanks @garnett! This is a great addition, so I'm merging it as-is. More fields can be added as necessary.

File: mas-cli/Commands/Info.swift (hunk 1 above)
Comment (phatblat) [reply mas-cli/Commands/Info.swift]: This is a great discussion, but for now, I've merged this PR as it stands. @Aghassi if you'd still like `AppInfoFormatter` to be split out can you open a new issue or PR?

**Pull request merged**